app/api/aact2/v1/app_api.rb

```diff
@@ -0,0 +1,44 @@
+module AACT2
+  module V1
+    class AppAPI < Grape::API
+      desc 'app status' do
+        detail 'this returns a health status'
+        named 'app_storage'
+        failure [
```

Comment:
this is a peculiarity with how grape_swagger generates documentation for the endpoint in swagger. It uses the failure array of arrays to generate all of the possible return codes. I figured this out when we did the Duke Data Service.  It generates the '[Response Messages](https://api.dataservice.duke.edu/apiexplorer#!/current_user/GET_api_version_current_user_format)' from it, which should include 200 (or 204, etc.).